db/executer.go

```diff
@@ -58,6 +61,11 @@ func (b *dbDecorator) find(out *[]Record, limit int, where ...interface{}) error
 	return db.Error
 }
 
+func (b *dbDecorator) getList(out *[]BlackDevice) error {
```

Comment:
I think `find()` could be made more generic to allow for getting the blacklist, but I'm not very passionate about that.  At least, I think `find` should become `findRecords` and `getList` should be `findBlacklist` so we don't have find and get going on.